### ISSUE TITLE
Not compatible with new databases

### ISSUE THREAD

**Issue by conradtchan** [conradtchan/starfit#15]
@2sn reported that the new databases did not run at starfit.org. A more verbose error message shows that this is a problem with starfit, not the webpage.

```
result = starfit.Single(
File "/usr/local/lib64/python3.10/site-packages/starfit/fit.py", line 37, in __init__
self._setup(
File "/usr/local/lib64/python3.10/site-packages/starfit/starfit.py", line 72, in _setup
db = StarDB(db, silent=self.silent)
File "/usr/local/lib64/python3.10/site-packages/starfit/dbtrim.py", line 9, in __init__
super().__init__(*args, **kwargs)
File "/usr/local/lib64/python3.10/site-packages/starfit/autils/stardb.py", line 277, in __init__
self._from_file(**kw)
File "/usr/local/lib64/python3.10/site-packages/starfit/autils/stardb.py", line 509, in _from_file
s = f"File size: {byte2human(self.filesize)}"
AttributeError: 'TrimDB' object has no attribute 'filesize'
```

**Comment by 2sn** [comment]
@conradtchan what is the detailed error message?  As I mentioned, maybe stadb.py needs to be updated?

**Comment by conradtchan** [comment]
@2sn this is the error from the webpage and from running the starfit python package:
```
result = starfit.Single(
File "/usr/local/lib64/python3.10/site-packages/starfit/fit.py", line 37, in __init__
self._setup(
File "/usr/local/lib64/python3.10/site-packages/starfit/starfit.py", line 72, in _setup
db = StarDB(db, silent=self.silent)
File "/usr/local/lib64/python3.10/site-packages/starfit/dbtrim.py", line 9, in __init__
super().__init__(*args, **kwargs)
File "/usr/local/lib64/python3.10/site-packages/starfit/autils/stardb.py", line 277, in __init__
self._from_file(**kw)
File "/usr/local/lib64/python3.10/site-packages/starfit/autils/stardb.py", line 509, in _from_file
s = f"File size: {byte2human(self.filesize)}"
AttributeError: 'TrimDB' object has no attribute 'filesize'
```

**Comment by 2sn** [comment]
This has been fixed by #21
